File: Unidad2/MecanismosOrdenacion/BubbleSort.java

public class BubbleSort {
    public static void main(String[] args) {

        int[] calificaciones = {75, 90, 60, 85, 70};
        System.out.println("Calificaciones antes de ordenar:");

        for (int nota : calificaciones) {
            System.out.print(nota + " ");
        }
        // Bubble Sort
        for (int i = 0; i < calificaciones.length - 1; i++) {
            for (int j = 0; j < calificaciones.length - 1 - i; j++) {
                if (calificaciones[j] > calificaciones[j + 1]) {
                    int auxiliar = calificaciones[j];
                    calificaciones[j] = calificaciones[j + 1];
                    calificaciones[j + 1] = auxiliar;
                }
            }
        }
        System.out.println("\nCalificaciones ordenadas:");
        for (int nota : calificaciones) {
            System.out.print(nota + " ");
        }
    }
}

File: Unidad2/MecanismosOrdenacion/BubbleSort.py
calificaciones = [75, 90, 60, 85, 70]

print("Antes de ordenar:")
print(calificaciones)

# Bubble Sort
for i in range(len(calificaciones) - 1):

    for j in range(len(calificaciones) - 1 - i):

        if calificaciones[j] > calificaciones[j + 1]:

            auxiliar = calificaciones[j]
            calificaciones[j] = calificaciones[j + 1]
            calificaciones[j + 1] = auxiliar

print("Después de ordenar:")
print(calificaciones)